common/preamble.tex

```diff
@@ -126,13 +126,6 @@
 \lfoot{\today}
 \rfoot{Page \thepage\ of \pageref{LastPage}}
 
-%%%%
-%% Watermark  
-%%%%
-\usepackage[firstpage]{draftwatermark} % to remove replace `firstpage` with `nostamp`
-\SetWatermarkScale{5.0}
-
-
```

Comment:
Why are we removing this? If you do not want the watermark to show, use the instructions in the comment.

Reply:
It's unnecessary because there is already the text `--DRAFT--` at the top of every page.   Also, it was too big to fit on the page.   You could only see part of R, the full A and part of F, previously.  When I shrank it, it just looked bad and distracting (IMHO).

Reply:
I'm not too attached to this I just found the text easier to read without the watermark.  If the DRAFT watermark is convention or something let me know...